packages/modals/src/styled/StyledClose.ts

```diff
@@ -5,89 +5,26 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
-import {
-  getColorV8,
-  retrieveComponentStyles,
-  DEFAULT_THEME,
-  focusStyles
-} from '@zendeskgarden/react-theming';
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { IconButton } from '@zendeskgarden/react-buttons';
```

Comment:
`import { StyledIconButton } from '../../../buttons/src/styled/StyledIconButton';`

Importing `StyledIconButton` using relative paths would allows us to override `data-garden-id` without causing breaking changes.

Is that an acceptable solution?